### ISSUE TITLE
how long to be let back into consensus?

### ISSUE THREAD

**Issue by mokhabarat** [pi-node/instructions#265]
hello

i ran out of space on my laptop so i deleted all blockchain data (about 100GB) I also cleaned out the files manually and reinstalled the node and docker (been running the node since Feb 2021) and now my up time is about 84% but my incoming and outgoing connections are still N/A so is the latest block number 
when will I be let back into the consensus? its been like that for around 30 days!

**Comment by parknochang** [comment]
Outging & incoming N/A is an error. Check docker PI-Consensus logs. ACTION in node app
After proceeding with "Remove all blockchain data", create a PI-Consensus Container again.

**Comment by Macromart** [comment]
Yes hello mokhabarat, I know your problem well. I was rocking it with 8 x outgoing and 64 incoming connections. Then the pi remuneration stopped for a week, so I did what you did and wiped it all and reinstalled. Now I'm up again running the node for 5 days 97% up time with zero incoming connections. Their AI heavily penalises you for uninstalling and reinstalling to fix their buggy software. I'll leave it for 10 days. If it doesn't get any better, I’ll get a different computer (why? Because they use your mac address or CPU serial to know its you.) and give the account to a friend. Cheesed off over this issue

**Comment by cheme75** [comment]
My 12 yr old i5 pc is hanging in considering it’s running another container for something else. Overall uptime is around 90%, in/out are usually 8 each or more. Supporting other nodes and yet bonus will sit at zero for 4-5 days at a time. Here or there I can understand, but it’s become far too often recently.

**Comment by cheme75** [comment]
FWIW, my drive appears to be filling up and locking up windows this past few days. After rebooting, cleaning out temp files, dumping recycle bin, restarting container goes ok at least for a while and I see I/O activity fairly quickly and within a few minutes pi app shows “catching up” just not sure this drive has enough free space to avoid crashing windows again and stay up long enough to get back to latest block.
